tests/testthat/setup-logger.R

```diff
@@ -0,0 +1 @@
+logger::log_threshold(logger::FATAL, namespace = "teal")
```

Comment:
I actually don't know how to turn off the logger. FATAL is just temporary that is sufficient for now.

Available options are:
- custom layout to be an empty string
- custom appender function to be NULL
- ...